src/documentation/reference/ReferenceNode.tsx

```diff
@@ -79,6 +81,12 @@ const ReferenceNode = ({
             behavior: prefersReducedMotion ? "auto" : "smooth",
           });
         }
+        setTimeout(() => {
+          setHighlighting(true);
+          setTimeout(() => {
+            setHighlighting(false);
+          }, 3000);
+        }, 300);
       }, 150);
```

Comment:
Let me know what you think about timings here. The design only had the short fade so far as I can tell, but we need to have it linger for a while to make the initial selection clear. 

Or perhaps it's supposed to remain highlighted until you interact with the toolbar? What do you think?

Reply:
Added a fade-in after discussion.

@microbit-robert, if this is otherwise OK, then I think we should review and merge as is, then I'll tweak based on feedback once we've heard from more folk.